README.rst

```diff
@@ -85,23 +98,35 @@ Building on Windows with conda-build
 ::
 
     set PYVER=<3.6 or 3.7>
+    set NUMPYVER=<1.16 or 1.17>
     conda create -n CBLD -q -y python=%PYVER% conda-build conda-verify vc vs2015_runtime vs2015_win-64
     conda activate CBLD
     git clone https://github.com/IntelPython/sdc.git
     cd sdc
-    conda build --python %PYVER% --override-channels -c numba -c defaults -c intel buildscripts\sdc-conda-recipe
+    # Build Numba
+    conda build --python %PYVER% --numpy %NUMPYVER% --output-folder <path_to_sdc>\numba_build -c numba -c defaults -c intel --override-channels buildscripts\numba-conda-recipe\recipe
+    # Build Intel SDC
+    conda build --python %PYVER% --numpy %NUMPYVER% -c <path_to_sdc>\numba_build -c numba -c defaults -c conda-forge --override-channels buildscripts\sdc-conda-recipe
 
 Building on Windows with setuptools
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 ::
 
-    conda create -n SDC -c numba -c defaults -c intel -c conda-forge python=<3.6 or 3.7> numba impi-devel pyarrow=0.15.0 arrow-cpp=0.15.0 scipy pandas boost
+    set PYVER=<3.6 or 3.7>
+    set NUMPYVER=<1.16 or 1.17>
+    conda create -n SDC -c numba -c defaults -c intel -c conda-forge python=%PYVER% numpy=%NUMPYVER% pandas=0.25.3 scipy pyarrow=0.15.1 tbb-devel llvmlite=0.31.0rc1=py*_0
     conda activate SDC
     git clone https://github.com/IntelPython/sdc.git
     cd sdc
     set INCLUDE=%INCLUDE%;%CONDA_PREFIX%\Library\include
     set LIB=%LIB%;%CONDA_PREFIX%\Library\lib
-    %CONDA_PREFIX%\Library\bin\mpivars.bat quiet
+    # Build Numba
+    git clone https://github.com/numpy/numpy.git
+    cd numba
+    git checkout ef119bcd1733ff49d71bdf2da8a66e91bb704f83
```

Comment:
This magic constant is commit number. Could you, please, create variable with name explaining what is specific with this commit?

Reply:
Do not think it is required to create a variable for checkouting Numba hash
I just mentioned  that
"Intel SDC uses Numba ``ef119bcd1733ff49d71bdf2da8a66e91bb704f83`` commit from master branch as Numba package for build and run.
That is why it is required to build specified Numba first. Build steps are described below."
Do you think there should be described features which we are using?